charts/mattermost-operator/templates/mattermost-operator/deployment.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.mattermostOperator.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "mattermost-operator.name" . }}
+spec:
+  replicas: {{ .Values.mattermostOperator.replicas }}
+  selector:
+    matchLabels:
+      name: {{ template "mattermost-operator.name" . }}
+  template:
+    metadata:
+      labels:
+        name: {{ template "mattermost-operator.name" . }}
+    spec:
+      containers:
+      - args:
```

Comment:
may this should be also as part of flags which we can pass

Reply:
Added in values.yaml ;)